src/shared/components/categoricalFilterMenu/CategoricalFilterMenu.tsx

```diff
@@ -48,8 +48,9 @@ export default class CategoricalFilterMenu extends React.Component<
     @computed get filterTypeDropdown() {
         return (
             <select
-                className="btn btn-default btn-sm"
+                className="form-control input-sm"
```

Comment:
@Rajat-Sirohi we should use bootstrap classes to create consistency.   Can you also apply similar to the inputs on the slider.  Also, see stacked checkbox style https://getbootstrap.com/docs/3.3/css/ instead of the <ul>

Reply:
`<ul />`